src/style.css

```diff
@@ -2,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.search__container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 100%;
+}
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it. If 'width: 100%' is not a requirement, consider removing it to allow the content to determine the size of the container.